app/controllers/hyrax/depositors_controller.rb

```diff
@@ -37,6 +37,10 @@ def destroy
 
     private
 
+    def sanitize_route_string(route)
```

Comment:
Could you add a comment about why this method? I think your PR helps give it context.  For those looking at it in the future, it would be helpful to give them insight into what's happening.